.github/ISSUE_TEMPLATE/membership.md

```diff
@@ -7,28 +7,32 @@ assignees: ''
 
 ---
 
+<!-- Please remember to change the title of this issue by replacing
+ <your-GH-handle> with the actual GitHub handle -->
+
 ### GitHub Username
 
 e.g. (at)example_user
 
-### Organization you are requesting membership in
-
-e.g. (at)open-telemetry
-
 ### Requirements
 
 - [ ] I have reviewed the community membership guidelines (https://github.com/open-telemetry/community/blob/master/community-membership.md)
 - [ ] I have enabled 2FA on my GitHub account. See https://github.com/settings/security
 - [ ] I have subscribed to the [Gitter channel](https://gitter.im/open-telemetry/community)
 - [ ] I am actively contributing to 1 or more OpenTelemetry subprojects
-- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
+- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirments, sponsors must be approvers or maintainers of at least one repository and not being affiliated with a single company
```

Comment:
```suggestion
- [ ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines. Among other requirements, sponsors must be approvers or maintainers of at least one repository in the organization and not both affiliated with the same company
```
Otherwise it sounds like one must have at least two jobs to be allowed to sponsor someone ;)